devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy

```diff
@@ -66,6 +66,7 @@ def installed() {
 		log.warn "Failed to add endpoint 2 ($desc) as Z-Wave Binary Switch Endpoint - $e"
 	}
 	configure()
+	response(refresh())
```

Comment:
Move response(refresh) to configure() where You are already calling refresh or remove refresh there and combine like Tom said. First way looks easier